README.md

```diff
@@ -20,7 +20,7 @@ You can now go to https://localhost
 What can I find in this demo? 
 =============================
 
-This demo application contains several things you may be interested.   
+This demo application contains several things you may be interested:
```

Comment:
```suggestion
This demo application contains several features you may be interested by:
```

Reply:
The preposition by after the adjective interested may be incorrect. Consider changing it to another preposition.

Reply:
In english, you are "interested **by**" something.

Reply:
> I still find weird to apply a filter to disable a behavior, instead of applying a filter to enable a feature.

I understand your POV, but it's an approach I find interesting (and used in real projects). Maybe, for the next example, we can do something more in the spirit of what show the documentation and the [RegexpFilter](https://api-platform.com/docs/core/filters/#creating-custom-doctrine-orm-filters). It will be complementary.

Reply:
> In english, you are "interested **by**" something.

Using "by" seems wrong in this context: https://english.stackexchange.com/q/556022/91524 🙂

Reply:
:+1: Yep, so maybe reformulating your sentence like `This demo application contains several features you may find interesting:`